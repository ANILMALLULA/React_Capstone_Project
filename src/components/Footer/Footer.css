.main-footer {
  display: flex;
  color: aliceblue;
  background-color: #424642;
  flex-wrap: wrap;
  padding: 2em;
  justify-content: "space-between";
  bottom: 0;
  width: 100%;
}
