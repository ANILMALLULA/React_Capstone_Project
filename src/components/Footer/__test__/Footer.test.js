import React from "react";
import ReactDOM from "react-dom";
import Footer from "../Footer";

import { render, cleanup } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";
import renderer from "react-test-renderer";

afterEach(cleanup);

it("renders without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(<Footer></Footer>, div);
});

it("renders footer crashing", () => {
  const { getByTestId } = render(<Footer></Footer>);
  expect(getByTestId("footer")).toHaveClass("main-footer");
});

it("renders child crashing", () => {
  const { getByTestId } = render(<Footer></Footer>);
  expect(getByTestId("footer").firstChild).toHaveClass("container");
});

it("matches snapshot", () => {
  const tree = renderer.create(<Footer />).toJSON();
  expect(tree).toMatchSnapshot();
});
