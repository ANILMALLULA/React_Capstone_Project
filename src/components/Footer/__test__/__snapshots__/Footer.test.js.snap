// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="main-footer"
  data-testid="footer"
>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="col-8"
      >
        <h5>
          Wipro Limited
        </h5>
        <h6
          className="list-unstyled"
        >
          <li>
            0123-456-789
          </li>
          <li>
            Pune, Maharastra
          </li>
          <li>
            INDIA
          </li>
        </h6>
      </div>
      <div
        className="col-4"
      >
        <h4>
          Contact details
        </h4>
        <ul
          className="list-unstyled"
        >
          <li>
            Anil Kumar M
          </li>
          <li>
            Andhra Pradesh
          </li>
          <li>
            123-456-7899
          </li>
        </ul>
      </div>
    </div>
    <hr />
    <div
      className="row"
    >
      <p
        className="col-sm"
      >
        Â©
        2021
         WIPRO LIMITED | All rights reserved
      </p>
    </div>
  </div>
</div>
`;
